### DISCUSSION TITLE
Welcome to epic-boardgame Discussions!

### DISCUSSION THREAD

**Discussion by gelila-s** [NorthTorontoChristianSchool/Tic-Tac-WHOA#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dvanterve** [comment]
Thanks Gelila, this could be a great tool!  I hadn't seen it yet.
